migrations.lua

```diff
@@ -0,0 +1,18 @@
+local schema = require("lapis.db.schema")
```

Comment:
Right now if you setup a new db via cloud.sql then it will still have the old column names. Should that file be updated as well or is the plan that when you create a new db you first set it up via cloud.sql and then run `lapis migrate`?

Reply:
The goal for this PR is to run cloud.sql then do lapis migrate.

In the future, I think the goal should be to start with lapis migrate.